bower.json

```diff
@@ -27,6 +27,13 @@
     "gold-email-input": "PolymerElements/gold-email-input#^0.9.0",
     "gold-zip-input": "PolymerElements/gold-zip-input#^0.9.0",
     "gold-cc-input": "PolymerElements/gold-cc-input#^0.9.0",
+    "gold-phone-input": "PolymerElements/gold-phone-input#^0.9.0",
     "gold-cc-expiration-input": "PolymerElements/gold-cc-expiration-input#^0.9.0"
+  },
+  "devDependencies": {
+    "paper-button": "PolymerElements/paper-button#^0.9.0",
+    "paper-dialog": "PolymerElements/paper-dialog#^0.9.0",
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#^0.9.0",
+    "paper-input": "PolymerElements/paper-input#^0.9.0"
```

Comment:
The Webcomponents Polyfill is being explicitly included, but does not appear here in the `devDependencies`.